pkgs/games/among-sus/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, port ? "1234" }:
+
+stdenv.mkDerivation {
+  pname = "among-sus";
```

From the [nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming): `If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".`

```suggestion
  pname = "among-sus-unstable";
```

---

This belongs to the version except if the package should be named ``amung-sus-unstable``.

---

I need a link or guideline for this at this point, since packages in nixpkgs do ... a lot of different things...

---

https://github.com/NixOS/nixpkgs/issues/68518

---

https://logs.nix.samueldr.com/nixos-dev/2020-10-30#1604088469-1604089015; leaving as is after discussion in #nixos-dev